pkg/config/validation/validation.go

```diff
@@ -3173,6 +3173,28 @@ func validateHTTPDirectResponse(directResponse *networking.HTTPDirectResponse) (
 	return
 }
 
+func validateHTTPMirrors(mirrors []*networking.HTTPMirrorPolicy) error {
+	errs := Validation{}
+	for _, mirror := range mirrors {
```

Comment:
can we test you have mirrors or mirror, not both?

Reply:
I think currently it is not prevented